### DISCUSSION TITLE
How do you increase the Home Assistant polling rate?

### DISCUSSION THREAD

**Discussion by DeFlanko** [timmo001/system-bridge#3022] [Q&A]
I've noticed in Home Assistant that the System Bridge polling rate is every 5 mins. Can this be decreased to near real time to 5 secs?

**Comment by timmo001** [comment]
Not sure where you got this number from?

The integration is realtime(ish). System Bridge pushes any data change after scheduled timer of 30 seconds for most data, and on change for media updates.

If the integration disconnects, Home Assistant will attempt to reconnect every 30 seconds until the websocket connection is reestablished

**Reply by DeFlanko** [reply]
Ok, Partial answer -- can there be specific timers for specific sensors like CPU's and GPU's to be ever quicker? I ask because im looking for Operational values not Analytical values.

**Comment by DeFlanko** [comment]
The Graph numbers only reflect values of every 5 mins. That graph values are what other visuals also use...for example `custom:bignumber-card` and `custom:mini-graph-card`

![image](https://github.com/timmo001/system-bridge/assets/5233969/4db78298-3456-4f10-9801-e4719ce32fa9)

![image](https://github.com/timmo001/system-bridge/assets/5233969/ee5da669-1ef1-4e8e-b919-8e961a040dc8)

**Comment by DeFlanko** [comment]
what part im not sure you have control over is what the Home Assistant integration is polling timing is or if that's a system wide setting...
